### PR TITLE
chore: add `@maschad` to codeowners

### PR DESCRIPTION
**Pull request by danielbate**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @digorithm @arboleya @Torres-ssf @Dhaiwat10 @danielbate @nedsalk @petertonysmith94
+* @digorithm @arboleya @Torres-ssf @Dhaiwat10 @danielbate @nedsalk @petertonysmith94 @maschad
```
Comment (danielbate) [inline .github/CODEOWNERS:1]: @arboleya this is invalid until chad has org access

Comment (maschad) [reply .github/CODEOWNERS:1]: I have access now so this should be fine.

**Comment by petertonysmith94** [comment]
Should this be a `feat`? 🚀 

Welcome @maschad!

**Review by petertonysmith94 — APPROVED** [review]

**Review by Dhaiwat10 — APPROVED** [review]

**Comment by Dhaiwat10** [comment]
Welcome sir @maschad

**Comment by maschad** [comment]
Thanks all! 🚀

**Pull request merged**